diff --git "a/F:\\Green\\Language\\RS113\\source\\vcl\\Vcl.ExtCtrls.pas" "b/E:\\PBox\\src\\Vcl.ExtCtrls.pas"
index 17e3dc3..4acad43 100644
--- "a/F:\\Green\\Language\\RS113\\source\\vcl\\Vcl.ExtCtrls.pas"
+++ "b/E:\\PBox\\src\\Vcl.ExtCtrls.pas"
@@ -11578,23 +11578,34 @@ begin
   end;
 end;
 
+var
+  FCurrentSelectedCategoryPanel: TCustomCategoryPanel;
+
 procedure TCustomCategoryPanel.WMSetCursor(var Message: TWMSetCursor);
 var
   LMouseEvent: TTrackMouseEvent;
+  I: Integer;
 begin
   if Message.HitTest = HTNOWHERE then
     case Message.MouseMsg of
       WM_LBUTTONUP:
         begin
           FHeaderState := hsHot;
-          if FCollapsed then
-            Expand
+          for I := 0 to FCurrentSelectedCategoryPanel.PanelGroup.Panels.Count-1 do
+          begin
+            if FCurrentSelectedCategoryPanel <> TCategoryPanel( FCurrentSelectedCategoryPanel.PanelGroup.Panels.Items[I]) then
+             TCategoryPanel( FCurrentSelectedCategoryPanel.PanelGroup.Panels.Items[I]).Collapse;
+          end;
+
+          if FCurrentSelectedCategoryPanel.FCollapsed then
+            FCurrentSelectedCategoryPanel.Expand
           else
-            Collapse;
+            FCurrentSelectedCategoryPanel.Collapse;
           Exit;
         end;
       WM_LBUTTONDOWN:
         begin
+          FCurrentSelectedCategoryPanel := Self;
           FHeaderState := hsPressed;
           UpdateButtonState;
           if TabStop then
